printsorted/print_sorted.go

```diff
@@ -0,0 +1,28 @@
+package printsorted
+
+import (
+	"fmt"
+	"sort"
+)
+
+func PrintSorted(m map[int]string) {
+	for _, v := range sortMap(m) {
+		fmt.Print(v, " ")
+	}
+
+	fmt.Println()
+}
+
+func sortMap(m map[int]string) map[int]string {
+	r := map[int]string{}
+	k := make([]int, 0, len(m))
+	for index := range m {
+		k = append(k, index)
+	}
+	sort.Ints(k)
+	for _, index := range k {
+
+		r[index] = m[index]
+	}
+	return r
```

Comment:
The order of keys inside the `r` is not obvious. We can not say that keys will be sorted inside the map.

Reply:
Yes, forgot that in go `map` is not sorted. Fixed https://github.com/VSKut/go-temp/pull/6/commits/4bc454ca9bfc001df7bc3650341a5c4c565c265e

Reply:
Yeah, forgot that in go `map` is not sorted. Fixed https://github.com/VSKut/go-temp/pull/6/commits/4bc454ca9bfc001df7bc3650341a5c4c565c265e